spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/AbstractMessageListenerContainer.java

```diff
@@ -353,17 +345,12 @@ public void addQueues(Queue... queues) {
 	public boolean removeQueueNames(String... queueNames) {
 		Assert.notNull(queueNames, "'queueNames' cannot be null");
 		Assert.noNullElements(queueNames, "'queueNames' cannot contain null elements");
-		AtomicBoolean result = new AtomicBoolean();
 		if (this.queues.size() > 0) {
```

Comment:
Why do we still need this `if`? Won't `removeIf()` returns `false` if there is nothing removed

Reply:
If there are no queues, it avoids creating the set.

Reply:
But it's an unlikely  condition - feel free to change on merge.